extensions/functions_datetime.yaml

```diff
@@ -3,36 +3,111 @@
 scalar_functions:
   -
     name: extract
-    description: Extract portion of a date/time value.
+    description:  >-
+      Extract portion of a date/time value.
+      * YEAR Return the year of the timestamp.
+      * ISO_YEAR Return the ISO 8601 week-numbering year. First week of an ISO year has the majority (4 or more) of its
+        days in January.
+      * US_YEAR Return US epidemiological year. First week of US epidemiological year has the majority (4 or more) of
+        its days in January. Last week of US epidemiological year has the year's last Wednesday in it.
+        US epidemiological week starts on Sunday.
+      * QUARTER Return the quarter from a date argument (1-4).
+      * MONTH Return the month from the date passed (1-12).
+      * DAY Return the day of the month (1-31).
+      * DAY_OF_YEAR Return the day of year (1 to 366). January 1st maps to day number 1, February 1st to 32, etc.
+      * MONDAY_DAY_OF_WEEK Return the day of week where Monday (1) to Sunday (7).
+      * SUNDAY_DAY_OF_WEEK Return the day of week where Sunday (1) to Saturday (7).
+      * MONDAY_WEEK Return week of year number (1-53). First week starts on first Monday of January.
+      * SUNDAY_WEEK Return week of year number (1-53). First week starts on first Sunday of January.
+      * ISO_WEEK Return ISO week of year number (1-53). First ISO calendar week has the majority (4 or more) of its
+        days in January. ISO week starts on Monday.
+      * US_WEEK Return ISO-like week of year number (1-53). First US calendar week has the majority (4 or more) of its
+        days in January. US week starts on Sunday.
+      * HOUR Return the hour (0-23).
+      * MINUTE Return the minute (0-59).
+      * SECOND Return the second (0-59).
+      * MILLISECOND Return number of milliseconds since the last full second.
+      * MICROSECOND Return number of microseconds since the last full millisecond.
+      * SUBSECOND Return number of microseconds since the last full second of the given timestamp.
+      * UNIX_TIME Return number of seconds that have elapsed since 1970-01-01 00:00:00 UTC, ignoring leap seconds.
+      * TIMEZONE_OFFSET Return number of seconds of timezone offset to UTC.
+
+      Timezone strings must be as defined by IANA timezone database (https://www.iana.org/time-zones).
```

Comment:
This can be left for follow-up but I wonder if a "timezone string" is a data type of its own.